packages/react-rest/src/services/useDeleteOne/useDeleteOne.test.ts

```diff
@@ -130,15 +108,17 @@ describe("react-rest/services/useDeleteOne", () => {
       expect(result.current).toEqual([
         expect.any(Function),
         {
-          status: "success",
-          meta: undefined,
-          error: undefined,
-          isResolved: true,
-          isPending: false,
-          isRejected: false,
-          isRevalidating: false,
-          isStale: false,
-          isSuccess: true,
+          ["id"]: {
+            status: "success",
+            meta: undefined,
+            error: undefined,
+            isResolved: true,
+            isPending: false,
+            isRejected: false,
+            isRevalidating: false,
+            isStale: false,
+            isSuccess: true,
+          },
         },
```

Comment:
could you add a test case (can be in addition to this one) where we check deleting multiple things, just to confirm that the contextual meta is persisted?

ie. 

```
result.current[0]("id-1")
result.current[0]("id-2")
```
and then make sure the meta in `waitFor result.current[1]` object contains both states for `"id-1"` and `"id-2"`?

Reply:
and then probably same in `useUpdateOne.test.ts`